components/molecules/BenefitCard.js

```diff
@@ -1,20 +1,24 @@
+import { ButtonLink } from "../atoms/ButtonLink";
 import PropTypes from "prop-types";
 
 export function BenefitCard(props) {
+  function moreInfoHandler() {
+    window.location.assign(`/benefit/${props.id}`);
+  }
```

Comment:
Was this intended to be left in?

Reply:
No, I definitely just forgot to remove it.